test/extended/operators/olm.go

```diff
@@ -70,4 +72,18 @@ var _ = g.Describe("[Feature:Platform] OLM should", func() {
 			}
 		})
 	}
+
+	// OCP-21082 - Implement packages API server and list packagemanifest info with namespace not NULL
+	// author: bandrade@redhat.com
+	g.It("Implement packages API server and list packagemanifest info with namespace not NULL", func() {
+		msg, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("packagemanifest", "--all-namespaces", "--no-headers").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		for _, packageManifest := range strings.Split(msg, "\n") {
+			fields := strings.Fields(packageManifest)
+			if string(fields[0]) == " " && len(fields) < 4 {
```

Comment:
This checks that there's the expected number of fields - could we loop through them and verify that namespace is one of them instead?

Reply:
This was discussed previously on https://github.com/openshift/origin/pull/23772#discussion_r328294416 with @njhale.  I'm verifying the number of fields due to one of them is optional(displayName), so the best approach that I found is to check when the line has less than 4 fields if the first character of the line is blank, which would check that namespace field was not returned. Please let me know if you have any suggestions.

Reply:
What about something like this?

```go
var packageserverLine string
for _, line := range strings.Split(msg, "\n") {
  if strings.Contains(line, "packageserver") {
    packageserverLine = line
  }
}
if !strings.Contains(packageserverLine, "openshift-operator-lifecycle-manager") {
  e2e.Failf("bad")
}
```

that way we look for a line we expect to be there, and ensure it has the correct value of the namespace?

Reply:
Awesome, that will make this test to perform faster. Thank you for the suggestion

Reply:
Changed as requested, I just changed the `packageserver` to `etcd` and namespace to `openshift-marketplace`. The test passed locally. Thanks